app/views/admin/bookings/index.html.erb

```diff
@@ -1,13 +1,38 @@
-  <% if @bookings.any? %>
-  <!-- Main content -->
-  <div class="col-12">
-    <div class="card">
-      <!-- /.card-header -->
-      <div class="card-body">
-        <form class="form-inline mb-3 w-100 col-12 d-flex flex-nowrap">
-          <input class="form-control mb-0 mr-5 flex-shrink-0" style="width:90%" type="search" placeholder="Search" aria-label="Search">
-          <button class="btn btn-primary my-2 mr-5 " style="height:37px" type="submit">Tìm kiếm</button>
-        </form>
+<!-- Main content -->
+<div class="col-12">
+  <h1>Danh sách các booking</h1>
+  <div class="card">
+    <!-- /.card-header -->
+    <div class="card-body">
+      <p>Filters: <span class="filters"></span></p>
+      <div class="btn-group">
+        <button type="button" class="btn btn-default dropdown-toggle">
+          <span class="glyphicon glyphicon-filter"></span>
+          <span class="sr-only">Filters</span>
+        </button>
+        <%= form_with(url: admin_bookings_path, method: "get") do |f| %>
+          <ul class="dropdown-menu dropdown-menu-filter">
+            <li>
+              <%= f.label :booking_date, "Booking date"%>
+              <%= f.date_field :booking_date, class: "form-control date-input", value: params[:booking_date] %>
+              <%= f.label :check_in, "Check in"%>
+              <%= f.date_field :check_in, class: "form-control", value: params[:check_in] %>
+              <%= f.label :check_out, "Check out"%>
+              <%= f.text_field :check_out, class: "form-control", value: params[:check_out] %>
+              <%= f.label :status, "Status"%>
+              <%= select_tag(:status, options_for_select([["all", ""],
```

Comment:
cái options for select e viết vào helper nhé
Dùng Booking.statuses.map {|status| [status, status]}
để tạo mảng thay vì thủ công ntn nhé
sau đó dùng unshift để chèn all vào đầu mảng